.github/CODEOWNERS

```diff
@@ -84,3 +84,10 @@
 # FSE Workflow Files
 /.github/workflows/editing-toolkit-plugin.yml @Automattic/cylon
 /.github/workflows/send-calypso-app-build-trigger.sh @Automattic/cylon
+
+# Experimentation platform legacy library and new client
+/client/lib/abtest @Automattic/experimentation-platform
+/client/components/data/query-experiments @Automattic/experimentation-platform
```

Comment:
@withinboredom I'm not sure if this covers everything. I just did a search for `experiment` and followed the paths from #39160.

Reply:
That looks like everything to me.